.npmignore

```diff
@@ -1,12 +0,0 @@
-/.npmignore
-/.travis.yml
-/AUTHORS
-/build
-/test
-/Changelog.txt
-/jsl.conf
-/release.sh
-
-/pkg
-!/pkg/sinon.js
-!/pkg/sinon-ie.js
```

Comment:
I don't have ruby installed, and I'd prefer not to. Should `pkg` be included as well?